src/app/globals.css

```
@@ -87,15 +87,71 @@
+    .request-grid {
+      gap: 1.25rem;
+    }
+  }
+
+  .request-card-media {
+    inset: -16px 0;
+    height: calc(100% + 32px);
+    transform: translate3d(
+        var(--request-card-hover-x, 0px),
+        calc(
+          var(--request-card-media-offset, 0px) +
```

Stylelint (17.11.1)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)